enclave-tls/src/verifiers/api/enclave_verifier_register.c

```diff
@@ -51,7 +41,7 @@ enclave_verifier_err_t enclave_verifier_register(const enclave_verifier_opts_t *
 
 	/* Default type equals to name */
 	if (new_opts->type[0] == '\0')
-		strcpy(new_opts->type, new_opts->name);
+		etls_strncpy(new_opts->type, new_opts->name, strlen(new_opts->name) + 1);
```

Comment:
If an adversary intends to pass a long new_opts->name, new_opts->type will be overflow. Please use snprintf to fix up this issue.